filter_plugins/config_encoders.py

```diff
@@ -519,8 +519,14 @@ def encode_json(
 
 
 def encode_logstash(
-        data, convert_bools=False, convert_nums=False, indent="  ", level=0,
-        prevtype="", section_prefix=":", backslash_escaping_ignore_string='@@@'):
+        data,
+        convert_bools=False,
+        convert_nums=False,
+        indent="  ",
+        level=0,
+        prevtype="",
+        section_prefix=":",
+        backslash_escaping_ignore_string='@@@'):
```

Comment:
Please could you keep the original format and just shorten the last parameter to `backslash_ignore_prefix` (and refactor the code accordingly)? Like that it will be under 80 characters.

Reply:
Sorry, I re-read your comment. I renamed the parameter as per your request. :sweat_smile: